doc/java-generation-from-CRD.md

```diff
@@ -117,6 +120,10 @@ Usage: java-gen [-hV] [-add-extra-annotations] [-enum-uppercase]
 And the corresponding configurations of the Maven plugin are (output of `mvn help:describe -DgroupId=io.fabric8 -DartifactId=java-generator-maven-plugin -Dversion=<version> -Ddetail`):
 
 ```
+    alwaysPreserveUnknown
+      User property: fabric8.java-generator.always-preserve-unknown
+      Always inject additional properties in the generated classes
```

Comment:
nit, although they both seem to mean the same thing, this description of property doesn't seem to match with `-always-preserve-unknown` command line argument. Is this intentional?

Reply:
aligned all of the descriptions/comments across cli/maven and gradle plugin